Tests/TuistCloudTests/ValueGraph/ValueGraphTests.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
```

Comment:
Should `ValueGraphTests.swift` be in `TuistCoreTests` ? Or were there going to be some `TuistCloud` specifics to test?

Reply:
😅 Yes... I'll move it